### PR TITLE
New package: RadioSky.Pipe version 2.7.34

### LINKED ISSUE #115540: [Package Request]: Radio-SkyPipe II
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://radiosky.com/skypipeishere.html
Publisher: Radio-Sky
Package Name: Radio-SkyPipe II
Description: An Internet Enabled Strip Chart Recorder
Package Version: 2.7.34
Installer URL: https://radiosky.com/skypipe/RSPII_Install_2_7_34.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve https://github.com/microsoft/winget-pkgs/issues/115540

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122787)

### CODE REVIEW

**Comment by Dragon1573** [comment]
This version of package is too old for modern OS. There's an information dialog during the installation process. It may block the process and encounter a `Validation-Unattended-Failed` label.

Snipaste_2023-10-16_00-00-58

I have to solution at the moment. Help needed. 😕

**Comment by Dragon1573** [comment]
See above comments:

- https://github.com/microsoft/winget-pkgs/pull/122787?notification_referrer_id=NT_kwDOAvoKlbM4MDYyMjg5MTM2OjQ5OTQxMTQx#issuecomment-1763436618

File: manifests/r/RadioSky/Pipe/2.7.34/RadioSky.Pipe.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PublisherUrl: https://www.radiosky.com/
+PublisherSupportUrl: https://radiosky.com/contact.html
+PrivacyUrl: https://radiosky.com/privacy_statement.html
+Author: Radio-Sky Publishing
+PackageName: Radio-SkyPipe
+PackageUrl: https://radiosky.com/skypipeishere.html
+License: Proprietary
+Copyright: Copyright (c) 2012 by Radio-Sky Publishing
+ShortDescription: Internet Enabled Strip Chart Recorder.
+Description: An Internet Enabled Strip Chart Recorder
```
Comment (Dragon1573) [inline manifests/r/RadioSky/Pipe/2.7.34/RadioSky.Pipe.locale.en-US.yaml:17]:
```suggestion
```

`Description` should be removed when it's the same as `ShortDescription`.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by Dragon1573** [comment]
While launching the installer on Windows 7 or newer system, it will prompt out a warning said that users should run a capabitily wizard or troubleshooter after the install and before their first launch.

This warning cannot be supressed. It will block the unattended installation process and prevent this package from passing the test.

For the above reason, this pull request will be closed without merged. Please keep this pull request associate with the issue to remind other contributers of this situation.

**Pull request closed without merging**